src/posix/platform/spi_interface.hpp

```diff
@@ -79,14 +79,14 @@ class SpiInterface
      *
      * @note This method should be called before reading and sending spinel frames to the interface.
      *
-     * @param[in]  aPlatformConfig  Platform configuration structure.
+     * @param[in]  aArguments         Arguments parsed from radio url.
      *
      * @retval OT_ERROR_NONE          The interface is initialized successfully.
      * @retval OT_ERROR_ALREADY       The interface is already initialized.
      * @retval OT_ERROR_INVALID_ARGS  The UART device or executable cannot be found or failed to open/run.
      *
      */
-    otError Init(const otPlatformConfig &aPlatformConfig);
+    otError Init(Arguments &aArguments);
```

Comment:
nit: const

Reply:
Since `Arguments` uses `strtok`, we cannot pass it as const here.